packages/fxa-auth-server/fxa-oauth-server/lib/routes/authorization.js

```diff
@@ -50,7 +51,7 @@ module.exports = {
         .uri({
           scheme: ALLOWED_SCHEMES,
         }),
-      scope: validators.scope,
+      scope: validators.scope.default(ScopeSet.fromArray([])),
```

Comment:
I think it would be reasonable to make `scope` required when granting an authorization code, but didn't want to risk breaking in some edge-case where we're not doing that currently.

Reply:
We require a scope of min length 1 on the front end, so my feeling is it should be safe to make required here too - see https://github.com/mozilla/fxa/blob/7228646ae64fb7ddce9071e567c94b1041382bcd/packages/fxa-content-server/app/scripts/models/reliers/oauth.js#L54:L56